message.go

```diff
@@ -123,30 +123,30 @@ func (f HeaderFlags) String() string {
 type HeaderType uint16
 
 const (
-	// HeaderTypeNoop indicates that no action was taken.
-	HeaderTypeNoop HeaderType = 0x1
+	// Noop indicates that no action was taken.
+	Noop HeaderType = 0x1
 
-	// HeaderTypeError indicates an error code is present, which is also
+	// Error indicates an error code is present, which is also
 	// used to indicate success when the code is 0.
-	HeaderTypeError HeaderType = 0x2
+	Error HeaderType = 0x2
```

Comment:
I think it's _probably_ okay to let this constant have the name `netlink.Error`, since we can always do `netlink.OpError` for the structure discussed in #126.

Reply:
I think so too.